Build Error on Rpi CM3 with Bullseye dist.

Hi,
I have followed all steps in building guide:
https://fledge-iot.readthedocs.io/en/latest/building_fledge/building_fledge.html

I am building on a Raspberry Pi Compute Module 3 with latest OS from Foundation: Bullseye 2021-10-30.
All necessary libraries are installed and OS is up to date and I tried several times but I get the following error while building the MAIN branch:

[...]
In file included from /usr/include/python3.9/pytime.h:6,
                 from /usr/include/python3.9/Python.h:85,
                 from /home/pi/Fledge/C/services/north-plugin-interfaces/python/python_plugin_interface.cpp:16:
/home/pi/Fledge/C/services/north-plugin-interfaces/python/python_plugin_interface.cpp: In function ‘uint32_t pl                                                         ugin_send_fn(PLUGIN_HANDLE, const std::vector<Reading*>&)’:
/usr/include/python3.9/object.h:633:41: error: expected ‘(’ before ‘PyType_HasFeature’
  633 | #define PyType_FastSubclass(type, flag) PyType_HasFeature(type, flag)
      |                                         ^~~~~~~~~~~~~~~~~
/usr/include/python3.9/longobject.h:15:9: note: in expansion of macro ‘PyType_FastSubclass’
   15 |         PyType_FastSubclass(Py_TYPE(op), Py_TPFLAGS_LONG_SUBCLASS)
      |         ^~~~~~~~~~~~~~~~~~~
/home/pi/Fledge/C/services/north-plugin-interfaces/python/python_plugin_interface.cpp:392:5: note: in expansion                                                          of macro ‘PyLong_Check’
  392 |  if PyLong_Check(pReturn)
      |     ^~~~~~~~~~~~
make[3]: *** [C/services/north-plugin-interfaces/python/CMakeFiles/north-plugin-python-interface.dir/build.make                                                         :95: C/services/north-plugin-interfaces/python/CMakeFiles/north-plugin-python-interface.dir/python_plugin_inter                                                         face.cpp.o] Error 1
make[3]: Leaving directory '/home/pi/Fledge/cmake_build'
make[2]: *** [CMakeFiles/Makefile2:1043: C/services/north-plugin-interfaces/python/CMakeFiles/north-plugin-pyth                                                         on-interface.dir/all] Error 2
make[2]: Leaving directory '/home/pi/Fledge/cmake_build'
make[1]: *** [Makefile:149: all] Error 2
make[1]: Leaving directory '/home/pi/Fledge/cmake_build'
make: *** [Makefile:244: c_build] Error 2

--- a/C/services/north-plugin-interfaces/python/python_plugin_interface.cpp
+++ b/C/services/north-plugin-interfaces/python/python_plugin_interface.cpp
@@ -389,7 +389,7 @@ uint32_t plugin_send_fn(PLUGIN_HANDLE handle, const std::vector<Reading *>& read
 	}
 
 	// Check return type
-	if PyLong_Check(pReturn)
+	if(PyLong_Check(pReturn))
 	{
 		data = (long)PyLong_AsUnsignedLongMask(pReturn);	
 	}

